x-pack/plugins/ml/public/application/components/custom_urls/custom_url_editor/utils.ts

```diff
@@ -289,7 +362,7 @@ export async function getTestUrl(job, customUrl) {
   let testUrl = customUrl.url_value;
 
   // Query to look for the highest scoring anomaly.
-  const body = {
+  const body: any = {
```

Comment:
if I'm not mistaken `QueryDslQueryContainer` should work 
```suggestion
  const body: QueryDslQueryContainer = {
```

Reply:
Updated type in 642ce6dbe70bb719b6416e5a61184a6b966b71d7